src/api/journalEntries/journalEntry.ts

```diff
@@ -177,8 +177,21 @@ export const useUpdateJournalEntry = () => {
   const queryClient = useQueryClient();
 
   return useMutation(updateJournalEntry, {
-    onSuccess: async () => {
-      await queryClient.invalidateQueries("journalEntries")
+    onSuccess: async (newEntry) => {
+      await queryClient.fetchQuery(["journalEntries"], async () => {
+        const { data } = await API.query<any>({
+            query: journalEntries,
+            variables: { authorId: newEntry.authorId }
+        });
+
+        return data.journalEntries;
+      })
+      queryClient.setQueryData(["journalEntries"], (oldData: any) => {
+        if (oldData.length) {
```

Comment:
This is not completed. Doesn't do anything if they user doesn't have any entries yet.